opae.spec

```diff
@@ -79,15 +75,49 @@ This package contains OPAE tests.
 rm -rf mybuild
 mkdir mybuild
 cd mybuild
-%cmake .. -DBUILD_ASE=OFF
+%cmake .. -DBUILD_ASE=OFF -DOPAE_INSTALL_RPATH=OFF
 make -j
 
 %install
+mkdir -p %{buildroot}%{_datadir}/opae
+cp RELEASE_NOTES.md %{buildroot}%{_datadir}/opae/RELEASE_NOTES.md
+
+mkdir -p %{buildroot}%{_usr}/src/opae/cmake/modules
+for s in FindDBus.cmake \
+         FindGLIB.cmake \
+         FindOPAE.cmake \
+         FindQuartus.cmake \
+         FindQuesta.cmake \
+         FindRT.cmake \
+         FindUUID.cmake \
+         FindVerilator.cmake \
+         Findjson-c.cmake \
+         cmake_useful.cmake \
+         compiler_config.cmake \
+         libraries_config.cmake
+do
+  cp "cmake/modules/${s}" %{buildroot}%{_usr}/src/opae/cmake/modules
```

Comment:
why `cp` and not `install`?

Reply:
No particular reason, though I did see the permissions being changed by rpmbuild, and I don't understand why.  Setting the permissions with chmod after copying the files had no affect.